cheatsheets/gleam-for-elm-users.md

```diff
@@ -729,6 +733,85 @@ The standard library provides the [gleam/result](https://hexdocs.pm/gleam_stdlib
 Gleam has a `try` keyword that allows for early exit from a block if a `Result` is an error. The equivalent in Elm would require the use of `Result.andThen`. The `try` keyword in Gleam provides syntactic sugar which simplifies functions that handle results.
 
 
+## If expressions
+
+#### Elm
+
+Elm has syntax for if-expressions for control flow based on boolean values.
+
+```elm
+description =
+    if value then
+        "It's true!"
+    else
+        "It's not true."
+```
+
+#### Gleam
+
+Gleam has no built in if-expression syntax and instead relies on matching on boolean values in case-expressions to provide this functionality:
```

Comment:
built in -> built-in

(to be consistent with how you used that above at least)